### ISSUE TITLE
got multiple values for keyword argument 'arkit'

### ISSUE THREAD

**Issue by sayduck-daniel** [kcoley/gltf2usd#80]
Just installed this on AWS EC2 instance and I get this:

``` bash
Traceback (most recent call last):
  File "/usr/local/gltf2usd/Source/gltf2usd.py", line 728, in <module>
    convert_to_usd(args.gltf_file, args.usd_file, args.fps, args.scale, args.arkit, args.verbose, args.use_euler_rotation)
  File "/usr/local/gltf2usd/Source/gltf2usd.py", line 699, in convert_to_usd
    success = check_usd_compliance(resolved_asset, arkit=args.arkit)
  File "/usr/local/gltf2usd/Source/gltf2usd.py", line 662, in check_usd_compliance
    checker = UsdUtils.ComplianceChecker(rootLayer, arkit=arkit, skipARKitRootLayerCheck=False)
TypeError: __init__() got multiple values for keyword argument 'arkit'
```

Any idea from where this could come?

I'm building USD like this:

    $ python /usr/local/src/USD/build_scripts/build_usd.py /usr/local/USD --python --no-imaging --no-usdview

**Comment by kcoley** [comment]
Hi @sayduck-daniel. This looks like an API change in the latest USD update for ‘UsdUtils.ComplianceChecker’. We would have to add a check to see which version of USD is used and switch to the new syntax.

**Comment by sayduck-daniel** [comment]
I figured this out as soon as I posted this issue ^^ 

They released v18.11 2 days ago. I installed from https://github.com/PixarAnimationStudios/USD/archive/v18.09.tar.gz and it works

**Comment by kcoley** [comment]
I made an update which should now work for v18.11 too: https://github.com/kcoley/gltf2usd/commit/63bad7f65714593b00cf08bd84738e8f0942ce3b
